### PR TITLE
CRM-14096 civicrm.config code doesn't do a good job when civicrm not in ...

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
...sites/all

### CODE REVIEW

File: civicrm.config.php.drupal [hunk 1]
```diff
@@ -86,7 +85,7 @@ function civicrm_conf_init() {
         echo "<a href=\"../install/index.php?mode=standalone\">click here</a>.\n";
```
Comment (eileenmcnaughton) [inline civicrm.config.php.drupal]: do we still need this standalone comment?

Comment (totten) [inline civicrm.config.php.drupal]: ^^ @cividesk

**Comment by eileenmcnaughton** [comment]
Hang fire on this - reviewing

**Comment by eileenmcnaughton** [comment]
OK - this is good to go - the file is now exactly the same as the d7 version of the file - which had improvements not ported back to d6, I added one minor tweak onto d7 too

**Pull request merged**